streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -1727,6 +1738,15 @@ public static Set<String> verifyTopologyOptimizationConfigs(final String config)
         return verifiedConfigs;
     }
 
+    /**
+     * Return configured KafkaClientSupplier
+     * @return Configured KafkaClientSupplier
+     */
+    public KafkaClientSupplier getKafkaClientSupplier() {
```

Comment:
Was this part of the KIP and do we need it? `StreamsConfig` is public API and we cannot just change it.

Also, the client supplier does not implement `Configurable` and thus does not need to be configured.

I am ok with reusing `getConfiguredInstance` but we would need a small change to the KIP.

Reply:
Sure. Will update the KIP.

Reply:
Thanks. Can you send a follow up email to the vote thread, calling out the change?

We don't need to re-vote but if anybody would have an objection, we need to give them a chance to race it.